src/jobs/BenchmarkJob.jl

```diff
@@ -336,7 +336,7 @@ function execute_benchmarks!(job::BenchmarkJob, whichbuild::Symbol)
                       results = run(benchmarks; verbose = true)
 
                       println("SAVING RESULT...")
-                      BenchmarkTools.save(\"$(benchresults)\", "results", results)
+                      BenchmarkTools.save(\"$(benchresults)\", results)
```

Comment:
The `benchresults` filename extension needs to be changed from `.jld` to `.json` [here](https://github.com/JuliaCI/Nanosoldier.jl/pull/38/files#diff-527b6d68e6ab87cdbc89aabe56b6391bR308).

Reply:
😱